app/src/main/java/com/google/samples/apps/sunflower/workers/SeedDatabaseWorker.kt

```diff
@@ -27,25 +27,27 @@ import com.google.samples.apps.sunflower.data.AppDatabase
 import com.google.samples.apps.sunflower.data.Plant
 import com.google.samples.apps.sunflower.utilities.PLANT_DATA_FILENAME
 
-class SeedDatabaseWorker(context: Context, workerParams: WorkerParameters) : Worker(context, workerParams) {
+class SeedDatabaseWorker(context: Context, workerParams: WorkerParameters) :
```

Comment:
Thanks for fixing the line length style issue!  Can you update to match the style guide?

```
class SeedDatabaseWorker(
    context: Context,
    workerParams: WorkerParameters
) : Worker(context, workerParams) {
```

Reply:
done